### PR TITLE
Admin Field

### PR DESCRIPTION
**Pull request by johnayinde**
added isAdmin field to entity included the value on the tokens that is sent during signin both for email signIn and OAuth signIn

### CODE REVIEW

**Review by PhilemonBrain — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

File: src/database/migration/1671705762857-userAdmin.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      `ALTER TABLE "user" ADD "isAdmin" character varying`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "user" ADD CONSTRAINT "UQ_b2033a3235871353c93700a0b60" UNIQUE ("isAdmin")`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "user" RENAME CONSTRAINT "UQ_b2033a3235871353c93700a0b60" TO "UQ_84f67c49f832f13e682ba6839f4"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "user" RENAME COLUMN "isAdmin" TO "signupToken"`,
```
Comment (SunmadeBabafemi) [inline src/database/migration/1671705762857-userAdmin.ts:34]: hey, why did you change the signUpToken column to isAdmin

Comment (johnayinde) [reply src/database/migration/1671705762857-userAdmin.ts:34]: Signup token was never in the user entity, It only exists in the OTP entity which is not included in  files I changed.

it might have been added due to one of those migration glitch
